cypress/e2e/communityTemplates.test.ts

```diff
@@ -31,13 +31,21 @@ describe('Community Templates', () => {
     //on empty
     cy.getByTestID('lookup-template-button').click()
     cy.getByTestID('notification-error').should('be.visible')
+    cy.getByTestID('notification-error--dismiss').click()
+    cy.getByTestID('lookup-template-input').type('{enter}')
+    cy.getByTestID('notification-error').should('be.visible')
+    cy.getByTestID('notification-error--dismiss').click()
 
     //lookup template errors on github folder
     cy.getByTestID('lookup-template-input').type(
       'https://github.com/influxdata/community-templates/tree/master/kafka'
     )
     cy.getByTestID('lookup-template-button').click()
```

Comment:
i'd be fine not testing this `click` and instead just submitting with `enter`, since the click functionality is thoroughly tested.